tests/Language/Haskell/Stylish/Tests.hs

```diff
@@ -47,10 +46,9 @@ case02 = withTestDirTree $ do
     actual <- format (Just $ ConfigPath "test-config.yaml") Nothing input
     actual @?= result
   where
-    input = "module Herp where\n data Foo = Bar | Baz"
+    input = "module Herp where\ndata Foo = Bar | Baz"
     result = Right [ "module Herp where"
-                   , "data Foo = Bar"
-                   , "  | Baz"
+                   , "data Foo = Bar | Baz"
```

Comment:
`case01` and `case02` in this suite test whether the `indent` value was correctly read from the config file (and applied during formatting), or whether the default 4 was used. This behavior is no longer tested, so maybe it would make sense to add fields to `Bar` and `Baz` constructors?

Reply:
Done!